### ISSUE TITLE
om staged-config doesn't pull all values for PAS

### ISSUE THREAD

**Issue by geofffranks** [pivotal-cf/om#380]
When running `om staged-config`, against the PAS tile, some things, like customized sidebar links for apps manager in the PAS tile are missing (there may be more). Can this be fixed so that `staged-config` pulls in any values from a tile that are changed from the defaults?

**Comment by jtarchie** [comment]
Can you specify what values are missing from the `staged-config` by their property name?

There are some values that get dropped unless you include `--include-credentials` because we wanted to ensure that credentials were explicitly requested.

**Comment by geofffranks** [comment]
`push_apps_manager_nav_links` is the name of the product installation-settings identifier of something we're missing, not sure what the full property name would be though.

It's using a `records` attribute for its value vs `value`, maybe that's part of it? The only other items using `records` are credentially - credhub encryption keys, and user-supplied SSL certs.

**Comment by jtarchie** [comment]
@geofffranks, good find!

Here's the hunting process I went through to validate this bug.
1. Pulled a config for `cf` in a deployed environment. I validated that `push_apps_manager_nav_links` is missing.
1. Pulled the JSON payload from the API endpoint for properties. The payload contained the values for product.
    <img width="723" alt="Screen Shot 2019-06-10 at 4 46 56 PM" src="https://user-images.githubusercontent.com/75184/59232114-5b0b3480-8ba0-11e9-833e-f771f5927f8e.png">
    You'll notice that the attributes within the element have `configurable: false` and the collection has `configurable: true`.
1. Validated the metadata has the `configurable: true` for the collection.

```yaml
   - _hack: null
  configurable: true
  default:
  - href: /marketplace
    name: Marketplace
  - href: https://docs.pivotal.io/pivotalcf/2-5/pas/intro.html
    name: Docs
  - href: /tools
    name: Tools
  name: push_apps_manager_nav_links
  property_blueprints:
  - name: name
    type: string
  - name: href
    type: string
  type: collection
```
   It appears that attributes don't have `configurable: false` explicitly set.

4. The WebUI allows you to set the configs, even though the API says you cannot configure them.
<img width="871" alt="Screen Shot 2019-06-10 at 4 49 57 PM" src="https://user-images.githubusercontent.com/75184/59232229-d66ce600-8ba0-11e9-9224-973c935fba2a.png">

Since the `staged-config` filters out everything that is not `configurable`, that is why you aren't seeing the property in the output.

I've reached out to the OpsManager team for advice to approach. I'll update this with the follow up.

**Comment by nbconklin** [comment]
Hi there - we've discussed with the OpsManager team and they are planning to backport the API fix from 2.3-2.7. Stay tuned for the OpsManager release notes for more information. Closing this issue. Thanks for reporting!
